components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -326,13 +326,9 @@ function StarcraftMatchGroupInput._matchWinnerProcessing(match)
 			match.winner = 0
 			match.resulttype = 'draw'
 		end
-		if
-			tostring(match.winner) == tostring(opponentIndex) or
-			match.resulttype == 'draw' or
-			opponent.score == bestof / 2
-		then
+		if tonumber(match.winner) == opponentIndex or match.resulttype == 'draw' then
```

Comment:
Could you add a newline here?

Reply:
```suggestion

		if tonumber(match.winner) == opponentIndex or match.resulttype == 'draw' then
```

Reply:
Looks like it didn't work?

Reply:
![Screenshot 2022-12-22 12 03 15](https://user-images.githubusercontent.com/75081997/209120752-137e0914-9fcc-4ac6-b24c-9975042caf53.png)
i see a blank line

Reply:
Apologies, I meant that the if line was too long

Reply:
ah

```suggestion
		if tonumber(match.winner) == opponentIndex or
			match.resulttype == 'draw' then
```